README.md

```diff
@@ -175,10 +175,10 @@ canvas.toDataURL.mockReturnValueOnce(
 
 ## Contributors
 
-* [@hustcc](https://github.com/hustcc)
-* [@jtenner](https://github.com/jtenner)
-* [@evanoc0](https://github.com/evanoc0)
+- [@hustcc](https://github.com/hustcc)
```

Comment:
Contributors  all is here, https://github.com/hustcc/jest-canvas-mock/graphs/contributors

Maybe a table is more beautiful.

Reply:
I can redo the contributors too. I just wanted to add the list to the readme. Is that okay?

Reply:
Of course that is okay!